cli/src/main/scala/org/apache/celeborn/cli/config/CliConfigManager.scala

```diff
@@ -27,8 +27,11 @@ import org.apache.celeborn.common.util.Utils
 case class CliConfig(cliConfigData: Map[String, String])
 
 object CliConfigManager {
-  val cliConfigFilePath: String =
-    s"${sys.env.getOrElse("EXAMPLE_CONF_DIR", sys.env("HOME"))}/celeborn-cli.conf"
+  val cliConfigFilePath: String = {
+    val basePath = sys.env.getOrElse("CELEBORN_HOME", sys.env("HOME"))
```

Comment:
`CELEBORN_HOME` would be set - so cli will try writing to $CELEBORN_HOME/celeborn-cli.conf : and fail, as it would usually be read-only.

Reply:
let me know what you think

Reply:
I think `$HOME` would be better though, since as @mridulm  pointed out this would be installed in read-only paths. I can change it to just be for $HOME. cc @FMX

Reply:
it will be a hidden file anyways

Reply:
This change looks good to me.